index.html

```diff
@@ -271,12 +279,12 @@ <h1 class="primary">About Me</h1>
 <footer class="form-cont" id="form-cont">
   <form class="form" action="https://formspree.io/f/xnqrevbr" method="post">
 
-    <h2 class="h2">I'm always<br> interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    <h2 class="h2">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
     <div class="input">
      <div class="inp ini">
       <input type="text" id="name" placeholder="Full name" maxlength="30" required name="full name" class="inp1">
-      <!-- <input type="text" id="name" placeholder="First name" required maxlength="30" name="firstname" class="inp2">
-      <input type="text" id="name" placeholder="Last name" required maxlength="30" name="lastname" class="inp3"> -->
+      <input type="text" id="name" placeholder="First name" required maxlength="30" name="firstname" class="inp2">
+      <input type="text" id="name" placeholder="Last name" required maxlength="30" name="lastname" class="inp3">
```

Comment:
- [ ] Please make sure there is one input for Full name (just like the mobile version) and this should take the space of both First Name and Last Name inputs (shown in the desktop version). you should have one Input for full name just like mobile version 
currently we have two seperate input fields :point_down: 
![image](https://user-images.githubusercontent.com/97521557/206563716-3bf170b3-0fe3-4bb4-9cab-242f0a77b0de.png)